### PR TITLE
[Backport to v0.6] Upgrade rabbitmq to v3.8.9

### PR DESCRIPTION
**Pull request by atsikham**
Task #1935. There are other tasks for backporting recent changes #2163 #2164 #2165 #2166

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/add-repositories.sh [hunk 1]
```diff
@@ -12,8 +12,8 @@ echo "deb https://packages.grafana.com/oss/deb stable main" | tee /etc/apt/sourc
 wget -qO - https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
 echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list
 
-wget -qO - https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc | apt-key add -
-echo "deb http://dl.bintray.com/rabbitmq-erlang/debian bionic erlang-21.x" | tee /etc/apt/sources.list.d/erlang-21.x.list
+wget -qO - https://dl.bintray.com/rabbitmq/Keys/rabbitmq-release-signing-key.asc | apt-key add -
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/add-repositories.sh:15]: this should be mentioned in bintray task to fix it soon

**Review by mkyc — DISMISSED** [review]

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/add-repositories.sh (hunk 1 above)
Comment (atsikham) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/add-repositories.sh:15]: There is a separate task #2163 for that

**Review by przemyslavic — APPROVED** [review]

**Review by mkyc — APPROVED** [review]

**Pull request merged**